assets/scss/components/_summary-list.scss

```diff
@@ -0,0 +1,33 @@
+.govuk-summary-list--custom .govuk-summary-list__row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  h3 {
+    color:  $govuk-secondary-text-colour;
+  }
+  .govuk-summary-list__key {
+    flex: 1;
+  }
+  .govuk-summary-list__value {
+    flex: 2;
+  }
+  .govuk-summary-list__actions {
+    flex: 0 0 auto;
+  }
+  @media (max-width: 640px) {
+    .govuk-summary-list__key,
+    .govuk-summary-list__value,
+    .govuk-summary-list__actions {
+      flex: 1 1 100%;
+      text-align: left;
+      margin-bottom: 10px;
+    }
+    .govuk-summary-list__actions {
+      text-align: right;
+    }
+  }
+}
+
+h2.govuk-summary-card__title {
+  font-size: 2rem;
```

Comment:
The CSS should be as minimal as possible as GOVUK guideline is responsible of it. 
- If we want to override fonts size, they have created a class for it that you can find [here](https://design-system.service.gov.uk/styles/font-override-classes/). In our situation, a style has already been defined inside the summary-cards [here](https://design-system.service.gov.uk/components/summary-list/) and I think that we should be matching what is already existing `<h2 class="govuk-summary-card__title">` inside the html rather than in the css.
- For the grey colour, it should be applied for secondary texts rather than titles. To mark the difference in between titles, you can use [headings](https://design-system.service.gov.uk/styles/headings/) size in priority and if really needed looking for a contrasted colour, maybe like the brand colour (blue), but only if needed.